app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,10 @@
+.fullWidth {
+   width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered margin-left, margin-right, max-width, width